### PR TITLE
Ask for api key and secret when importing nodes from Ops Tools

### LINKED ISSUE #1684: Connect to Ops Tools using API key and secret
_To be initially delivered to OpsTool feature branch_

For software support we will need to store API key and secret to connect using basic auth  (instead of the key + url initially used in first development stage)

### PR DESCRIPTION
**Pull request by lquintai**
closes #1684 

User will be asked to provide a url, api key and api secret.

If unable to connect, user will be asked if they want to provide a CA certificate chain or to proceed insecurely.

Also fixes the orderer selection: if the user selects any orderer node, all orderer nodes will have hidden = false. 

Signed-off-by: Leonor Quintais <lquintai@uk.ibm.com>

### CODE REVIEW

File: packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts [hunk 1]
```diff
@@ -20,12 +20,13 @@ import { FabricEnvironmentRegistryEntry } from '../registries/FabricEnvironmentR
...
 import * as path from 'path';
 import { SettingConfigurations } from '../../configurations';
-import { FabricNode } from '../fabric/FabricNode';
+import { FabricNode, FabricNodeType } from '../fabric/FabricNode';
 import { FabricEnvironment } from '../fabric/FabricEnvironment';
 import { ExtensionCommands } from '../../ExtensionCommands';
 import { FileSystemUtil } from '../util/FileSystemUtil';
 import Axios from 'axios';
 import { ExtensionUtil } from '../util/ExtensionUtil';
+import https = require('https');
```
Comment (cazfletch) [inline packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts]: this should be imported like other node modules

File: packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts [hunk 2]
```diff
@@ -117,13 +117,51 @@ export async function importNodesToEnvironment(environmentRegistryEntry: FabricE
...
+                            const browseOptions: vscode.OpenDialogOptions = {
+                                canSelectFiles: true,
+                                canSelectFolders: false,
+                                canSelectMany: false,
+                                openLabel: 'Select',
+                                filters: {
+                                    Certificates: ['pem']
+                                }
+                            };
+                            const certificatePath: vscode.Uri = await UserInputUtil.browse('Enter a file path to the collection configuration', quickPickItems, browseOptions, true) as vscode.Uri;
```
Comment (toggle007) [inline packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts]: Perhaps we should change the message shown to the user (as they aren't entering a file path, but choosing a file instead?) to something like:
`Select the certificate (pem) you wish to add`

Comment (lquintai) [reply packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts]: Well spotted :)

File: packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts (hunk 1 above)
Comment (lquintai) [reply packages/blockchain-extension/extension/commands/importNodesToEnvironmentCommand.ts]: Addressed.

**Pull request merged**